packages/data-stores/src/plans/constants.ts

```diff
@@ -42,7 +53,29 @@ export const monthlySlugs = [
 	PLAN_ECOMMERCE_TRIAL_MONTHLY,
 ] as const;
 
-export const plansProductSlugs = [ PLAN_FREE, ...annualSlugs, ...monthlySlugs ] as const;
+export const annualSlugs = [ PLAN_PERSONAL, PLAN_PREMIUM, PLAN_BUSINESS, PLAN_ECOMMERCE ] as const;
+
+export const biennialSLugs = [
+	PLAN_PERSONAL_2_YEARS,
+	PLAN_PREMIUM_2_YEARS,
+	PLAN_BUSINESS_2_YEARS,
+	PLAN_ECOMMERCE_2_YEARS,
+] as const;
+
+export const triennialSlugs = [
+	PLAN_PERSONAL_3_YEARS,
+	PLAN_PREMIUM_3_YEARS,
+	PLAN_BUSINESS_3_YEARS,
+	PLAN_ECOMMERCE_3_YEARS,
+] as const;
+
+export const plansProductSlugs = [
```

Comment:
Needs the WooCommerce store slugs. Just adding any remaining slugs here should suffice - the resolver will normalize and populate the `PlanProduct` structure.